<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UTCameraWindow</class>
 <widget class="QMainWindow" name="UTCameraWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ImageWidget" name="bigImage" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cameraLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>500</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No Camera</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="topButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>500</y>
      <width>103</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Top</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bottomButton">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>500</y>
      <width>103</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Bottom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>500</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>601</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="labelLayout"/>
   </widget>
   <widget class="QPushButton" name="getButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>30</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>500</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copyRButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>500</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copyWButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>500</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy Sent</string>
    </property>
   </widget>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionNew_Table">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionYUV">
   <property name="text">
    <string>YUV</string>
   </property>
  </action>
  <action name="actionLAB">
   <property name="text">
    <string>LAB</string>
   </property>
  </action>
  <action name="actionHSI">
   <property name="text">
    <string>HSI</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>ImageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
